hadoop-tools/hadoop-azure/dev-support/testrun-scripts/cronjob.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+branchNames=$1
+IFS=',' read -r -a branches <<< "$branchNames"
+
+for branch in "${branches[@]}"
+do
+  echo "Branch: $branch"
+  git checkout "$branch"
+  git pull origin "$branch"
+  bash "PATH_TO_RUNTESTS_SCRIPT" "$branch"
```

Comment:
Where is the variable `PATH_TO_RUNTESTS_SCRIPT` set??

Reply:
It would be exported from the cronjob only

Reply:
My bad- it should have been "$PATH_TO_RUNTESTS_SCRIPT" instead. Corrected now.